components/badges/badges.css

```diff
@@ -0,0 +1,65 @@
+.profile-badge {
+    position: absolute;
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.5rem;
+    right: 0.5rem;
+    border: 2px solid var(--bg-color);
+    border-radius: 2rem;
+}
+
+.badge-container {
+    width: max-content;
+    position: relative;
+}
+
+.status-online {
+    background-color: var(--success-color)
+}
+
+.status-idle {
+    background-color: var(--warning-color);
+}
+
+.status-do-not-disturb{
+    background-color: var(--danger-color);
+}
+
+.status-offline {
+    background-color: var(--secondary-color);
+}
+
+.status-invisible {
+    background-color: var(--bg-color);
+    border: 3.5px solid var(--secondary-color);
+}
+
+.badge-status.badge-notification {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    border: none;
+    left: 60%;
+    bottom: 65%;
+    font-size: 0.9rem;
```

Comment:
If anything is less than 1rem then use pixel units

Reply:
will change that to px